filters/exclusions.txt

```diff
@@ -149,6 +149,8 @@ $popup,domain=aflamsexnek.com|
 /click/*$subdocument
 =adblock_
 ://partner.$third-party
+! https://github.com/AdguardTeam/FiltersRegistry/issues/392
+/^(https?|wss?):\/\/([0-9a-z\-]+\.)?([0-9a-z\-]+\.)(accountant|
```

Comment:
I guess you missed `/` at the end.

Reply:
this is not regexp, just a part of the rule